app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float,
+                 clean_power: int, average_rating: float,
+                 count_of_ratings: int) -> None:
+        if distance_from_city_center <= 0:
+            raise ValueError(
+                "Distance from city center must be greater than 0"
+            )
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        result = (car.comfort_class * (self.clean_power - car.clean_mark)
+                  * self.average_rating / self.distance_from_city_center)
+        return round(result, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
```

Comment:
The logic here updates `car.clean_mark` to `self.clean_power` if it is less than `self.clean_power`. This is correct, but ensure that this is the intended behavior, as it will set the clean mark to the station's clean power regardless of the car's initial clean mark.